### PR TITLE
Hyundai Ioniq 2019 EV FW versions

### PR DESCRIPTION
**Pull request by Richileaks**
add fingerprint Hyundai Ioniq Facelift 2019 BEV (EV)

### CODE REVIEW

**Comment by pd0wm** [comment]
What is your dongle id?

**Comment by Richileaks** [comment]
Hi,
My Dongle ID: d0798a8dab800ef1

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -171,12 +171,15 @@ class Buttons:
 FW_VERSIONS = {
   CAR.IONIQ_EV_2020: {
     (Ecu.fwdRadar, 0x7d0, None): [
+      b'\xf1\000AEev SCC F-CUP      1.00 1.01 99110-G7000         ',
```
Comment (pd0wm) [inline selfdrive/car/hyundai/values.py:174]:
```suggestion
      b'\xf1\x00AEev SCC F-CUP      1.00 1.01 99110-G7000         ',
```

File: selfdrive/car/hyundai/values.py [hunk 2]
```diff
@@ -171,12 +171,15 @@ class Buttons:
...
       b'\xf1\x00AEev SCC F-CUP      1.00 1.00 99110-G7200         ',
     ],
     (Ecu.eps, 0x7d4, None): [
+      b'\xf1\000AE  MDPS C 1.00 1.01 56310/G7310 4APEC101',
```
Comment (pd0wm) [inline selfdrive/car/hyundai/values.py:178]:
```suggestion
      b'\xf1\x00AE  MDPS C 1.00 1.01 56310/G7310 4APEC101',
```

File: selfdrive/car/hyundai/values.py [hunk 3]
```diff
@@ -171,12 +171,15 @@ class Buttons:
...
       b'\xf1\x00AE  MDPS C 1.00 1.01 56310/G7560 4APEC101',
     ],
     (Ecu.fwdCamera, 0x7c4, None): [
+      b'\xf1\000AEE MFC  AT EUR LHD 1.00 1.03 95740-G2500 190516',
```
Comment (pd0wm) [inline selfdrive/car/hyundai/values.py:182]:
```suggestion
      b'\xf1\x00AEE MFC  AT EUR LHD 1.00 1.03 95740-G2500 190516',
```

**Pull request merged**